.kokoro/system-tests/mongodb.sh

```diff
@@ -22,33 +22,36 @@
 rm -rf */*.log
 rm -rf *-*.yaml
 
-export NODE_ENV=development
-export DATA_BACKEND="mongodb"
+# TODO(franzih): Make these work again
+exit 0
 
-# Configure gcloud
-export GCLOUD_PROJECT=nodejs-getting-started-tests
-export GOOGLE_APPLICATION_CREDENTIALS=${KOKORO_GFILE_DIR}/secrets-key.json
-gcloud auth activate-service-account --key-file "$GOOGLE_APPLICATION_CREDENTIALS"
-gcloud config set project $GCLOUD_PROJECT
+# export NODE_ENV=development
+# export DATA_BACKEND="mongodb"
 
-# Install Node dependencies
-yarn global add @google-cloud/nodejs-repo-tools
-cd github/nodejs-getting-started
+# # Configure gcloud
+# export GCLOUD_PROJECT=nodejs-getting-started-tests
+# export GOOGLE_APPLICATION_CREDENTIALS=${KOKORO_GFILE_DIR}/secrets-key.json
+# gcloud auth activate-service-account --key-file "$GOOGLE_APPLICATION_CREDENTIALS"
+# gcloud config set project $GCLOUD_PROJECT
 
-# Copy secrets into subdirectories
-find . -name package.json -maxdepth 2 -execdir sh -c "cp ${KOKORO_GFILE_DIR}/secrets-config.json config.json" \;
-find . -name package.json -maxdepth 2 -execdir sh -c "cp $GOOGLE_APPLICATION_CREDENTIALS key.json" \;
+# # Install Node dependencies
+# yarn global add @google-cloud/nodejs-repo-tools
+# cd github/nodejs-getting-started
```

Comment:
any need to comment these out since you `exit 0` up above?

Reply:
I'm hoping that the MongoDB sources get put back in. Then we need the test again. Yes I know, it's all under source control, but that's much harder to find for somebody no familiar with the test structure.

Reply:
Oh, I see, you asked about `#`. Yes, they make it more clear that nothing is happening here. It's hard to spot an early exit otherwise.